taichi/util/environ_config.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "taichi/common/util.h"
+#include <string>
+#include <cstdlib>
+
+TLANG_NAMESPACE_BEGIN
+
+static int get_environ_config(const std::string &name, int default_value = 0)
```

Comment:
Does it make sense to make an interface like `template <typename T> get_environ_config(const std::string &name, const T &default_value) { ... }`, and then specialize it for `int`?

Reply:
I guess use override functions instead of `T` is better idea, since the func `stoi` and `stof`.

Reply:
You can always use different conversion functions in different specializations. Or you can simply use `if constexpr`.